### PR TITLE
adding collection to service and fix update bug.

### PR DESCRIPTION
**Pull request by Olaniyan99**

### CODE REVIEW

File: src/learning-objects/learning-objects.controller.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class LearningObjectsController {
     @Patch()
     @HttpCode(201)
     async updateAll(@Body() learningObjectDto: LearningObjectDto[]): Promise<{ message: string }> { 
-        if (learningObjectDto.length != 5)
+        if (learningObjectDto.length < 1 || learningObjectDto.length > 5)
```
Comment (pzalep1) [inline src/learning-objects/learning-objects.controller.ts:24]: Why does this have to be less than 1 or greater than 5?

Comment (EVogel1999) [reply src/learning-objects/learning-objects.controller.ts:24]: ^
The error also still shows that it must contain 5 objects.

File: test_environment/globalConfig.json [hunk 2]
```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:54437/f512a1ef-110e-44ab-aae9-876d2d378eeb?"}
\ No newline at end of file
+{"mongoUri":"mongodb://127.0.0.1:62124/ffea01b1-34fd-4fe2-9c8c-0d8f5e0e5d58?"}
```
Comment (EVogel1999) [inline test_environment/globalConfig.json:1]: Sorry I don't quite understand what this is for.  Could you explain??  Thank you!

Comment (Olaniyan99) [reply test_environment/globalConfig.json]: The test runner will generate a new URI every time you run it

**Pull request merged**